principles/work_in_the_open.md

```diff
@@ -8,7 +8,7 @@
 
 So you *could* develop your code in private, and release in time for a [Beta](https://www.gov.uk/service-manual/agile-delivery/how-the-beta-phase-works) assessment, which (other factors aside) would be sufficient to pass.
 
-However we believe there are more benefits to making code public from the beginning, rather than starting private and releasing later. Here we cover the reasons why, and who has the authority to release your code.
+However we believe there are more benefits to making code public from the beginning, rather than starting private and releasing later, and [we're not the only ones](https://mojdigital.blog.gov.uk/2017/02/21/why-we-code-in-the-open/). Here we cover the reasons why, and who has the authority to release your code.
```

Comment:
It might be clearer to describe the link in the text, e.g.

> and we're not the only ones, for example, see this [blog post by the Ministry of Justice on why they code in the open](https://mojdigital.blog.gov.uk/2017/02/21/why-we-code-in-the-open/).